docs/next/package.json

```diff
@@ -72,4 +67,4 @@
     "remark-preset-prettier": "^0.4.1",
     "typescript": "^4.1.3"
   }
-}
+}
```

Comment:
Do we need a newline here?

Reply:
err seems related to https://github.com/npm/npm/issues/18545 which adds this newline after i ran yarn/npm install